src/core/a-entity.js

```diff
@@ -423,15 +425,15 @@ var proto = Object.create(ANode.prototype, {
   },
 
   removeComponent: {
-    value: function (name) {
+    value: function (name, force) {
```

Comment:
`removeComponent` on a default component is currently a no-op. I didn't want to re-implement all the `removeComponent` logic for `reload`, so I just added the flag. I agree that it's kind of counter-intuitive that you can't remove default components, so it might be a better approach to make `removeComponent` unconditionally remove components, and change the callers to not call `removeComponent` for default components if that is the desired behavior.